README.md

```diff
@@ -1,20 +1,22 @@
 ![Hydra - the open source data warehouse](.images/header.png)
 
+[Request Access](https://hydras.io/#early-access) - [Documentation](https://docs.hydras.io/getting-started/readme) - [Demo](https://www.youtube.com/watch?v=DD1oD1LWNOo) - [Website](https://hydras.io/)
+
 Data driven decisions at every stage - [Hydra](https://hydras.io/) is your open source data warehouse built on Postgres.
 
 Postgres is the foundation of the Modern Data Stack. Hydra's mission is to make data-driven decisions accessible at every stage. We built Hydra on an open Postgres foundation that's easy to use. Everything that works with Postgres works with Hydra.
 
-* [x] hosted postgres database - [docs](https://docs.hydras.io/getting-started/hydra-with-postgres)
-* [x] append-only columnar store - [docs](https://docs.hydras.io/features/columnar)
-* [x] external tables - [docs](https://docs.hydras.io/features/external_tables)
-* [x] postgres scheduler - [docs](https://docs.hydras.io/features/scheduler)
-* [ ] vectorized execution
-* [ ] updates and deletes for columnar store
-* [ ] separation of compute and storage
+* [x] 🗃 hosted postgres database - [docs](https://docs.hydras.io/getting-started/hydra-with-postgres)
+* [x] 📎 append-only columnar store - [docs](https://docs.hydras.io/features/columnar)
+* [x] 📊 external tables - [docs](https://docs.hydras.io/features/external_tables)
+* [x] 📅 postgres scheduler - [docs](https://docs.hydras.io/features/scheduler)
+* [ ] ☠️ vectorized execution
```

Comment:
This emoji kills me! We could use any of: 🐎 🏎️ ⏩ 💨 📈

Reply:
🐎, let's save 🏎️ for the Performance section header